src/main/kotlin/io/github/inductiveautomation/kindling/utils/TabStrip.kt

```diff
@@ -151,6 +177,32 @@ open class TabStrip : DnDTabbedPane() {
         )
     }
 
+    fun editTabTitle(tabIndex: Int) {
+        val textField = JTextField(
+            getTitleAt(tabIndex),
+        )
+
+        textField.addKeyListener(
+            object : KeyAdapter() {
+                override fun keyPressed(e: KeyEvent?) {
+                    if (e?.keyCode == KeyEvent.VK_ENTER) {
```

Comment:
Nit: Two (mutually exclusive) ifs in a row might as well be a `when`. Generally the rule is three, and it'd be totally fine to leave this as-is, but it's nice to lean on when/single expression outputs as much as possible.